assets/packages/limesurvey/survey.js

```diff
@@ -433,6 +433,9 @@ function activateSoftMandatory(){
 }
 
 function resetQuestionTimers(sid) {
+    if(!window.localStorage) {
+        return;
```

Comment:
By applying this fix, what the participant experience would be like?
They would be able to take the survey without the timed questions?

If a survye amdin expected timedquestions, would it be ok to allow participants not to use it?
What if we raise an alert if this is the case?

Reply:
To be honest, I have not tested the timers, the aim for me was to get past javascript crashing.

If the participant does not have localStorage, the questionTimers could not have worked in any case since it uses the localStorage based on what I can read from the code. So logically assuming such users would go from a javascript exception (and a lot of things not potentially workining) to silently question timers not working.

Reply:
> By applying this fix, what the participant experience would be like? They would be able to take the survey without the timed questions?

Before : all javascript are broken 

> If a survye amdin expected timedquestions, would it be ok to allow participants not to use it? What if we raise an alert if this is the case?


Before it don't work too, then it's a different (and new issue) see my comment here : https://github.com/LimeSurvey/LimeSurvey/pull/3856#discussion_r1611140732

Reply:
PS : i always think timer script must move to a core external plugin.

Reply:
I see that some of the changes need the webpack files to be rebuilt, I will check those also and do a commit

Reply:
ok, I changed return to `null` instead of `0` in `_getTimerFromLocalStorage`. With 0 it ended up going instantly forward. Also I built the webpack files. 

if one has a timer question with no localStorage, the question with Timer now looks like this: 
![image](https://github.com/LimeSurvey/LimeSurvey/assets/6357451/2f803066-bc61-47ee-9270-a8e40464b8eb)
and it will not trigger the timer.

without this fix, the page javascript throes exception:
![image](https://github.com/LimeSurvey/LimeSurvey/assets/6357451/5f0e4956-f7af-4d5f-8930-3f46391a3319)
the filter on page do not work, timer is not working, empty:
![image](https://github.com/LimeSurvey/LimeSurvey/assets/6357451/e4e27d8d-6aba-44ba-994c-e64941c37f94)

Reply:
> if one has a timer question with no localStorage, the question with Timer now looks like this: 

_now_ : you mean : current master ?

Reply:
I mean with THIS PR by "now"

Reply:
> I mean with THIS PR by "now"

OK then,

Without this PR
1. Timer didn't work
2. conditions didn't work

With this PR
1. Timer didn't work
2. conditions work